pain-points.md

```diff
@@ -0,0 +1,21 @@
+# Ouch
+
+## Stuff that are actually wrong
+
+- `store/home` is also defined in `vtex.store`, we should tell people they can extend that, but we maybe should also incentivize them to extend `vtex.dreamstore` whenever possible. What now?
+- It's hard to explain that we can extend a block `a` by declaring `a` in another file without the need to call it `a/b`.
```

Comment:
Like in the `vtex.shelf`/`partner.shelf` example?

Reply:
Yeap.